docs/DeveloperGuide.md

```diff
@@ -885,6 +961,10 @@ Similar to UC03
 * **GUI**: Graphical User Interface
 * **MSS**: Main Success Scenario
 * **API**: Application Programming Interface
+* **Time slots**: Hourly time intervals 
+* **Free time**: Unoccupied time intervals that a Person or a Group has
```

Comment:
Perhaps it would be better to change this to Free time slots instead of just Free time